xwayland/xwm.c

```diff
@@ -948,10 +970,11 @@ static void xwm_handle_map_request(struct wlr_xwm *xwm,
 	xsurface_set_wm_state(xsurface, XCB_ICCCM_WM_STATE_NORMAL);
 	xsurface_set_net_wm_state(xsurface);
 
-	uint32_t values[1];
-	values[0] = XCB_STACK_MODE_BELOW;
-	xcb_configure_window(xwm->xcb_conn, ev->window,
-			XCB_CONFIG_WINDOW_STACK_MODE, values);
+	xcb_configure_window(xwm->xcb_conn, ev->window, XCB_CONFIG_WINDOW_STACK_MODE,
```

Comment:
Maybe we can use `wlr_xwayland_surface_restack` instead of doing this?

Reply:
Sounds reasonable, done.